insta2/src/components/CommentForm.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'; 
+// import CommentSection from './CommentSection/CommentSection';
+
+class CommentForm extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+        comments: props.comments,
+        comment: ''
```

Comment:
There can be some pitfalls setting state with props, you may want to read the docs here https://reactjs.org/docs/react-component.html#constructor and read over the note on setting state with props